.rubocop.yml

```diff
@@ -172,3 +172,8 @@ Lint/ConstantOverwrittenInRescue: # new in 1.31
   Enabled: true
 Lint/NonAtomicFileOperation: # new in 1.31
   Enabled: true
+
+RSpec/Capybara/SpecificMatcher: # new in 2.12
+  Enabled: true
+RSpec/Rails/HaveHttpStatus: # new in 2.12
+  Enabled: true
```

Comment:
We're not using Rails in this gem, so maybe `false` would make the checks faster. 🤷

Reply:
good point!